jest.default.config.js

```diff
@@ -154,6 +154,7 @@ module.exports = {
     '**/integrations/**/*.(test).[j]s?(x)',
     '**/util/**/*.(test).[j]s?(x)',
     '**/*.(test).[j]s?(x)',
+    '**/*.(test).ts?(x)',
```

Comment:
I think this shouldn't be added here, as JS-code/repl will not be able to understand typescript conventions & I think that is the reason why cdk tests(all ts ones) are not passing.